conversationstarters.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'starter.json'), 'r', encoding="utf8") as f:
+    starters = json.load(f)
+
+
+class ConvoStarters(commands.Cog):
+    """A cog which posts easter conversation starters"""
```

Comment:
We know it's a cog, so this can be kept to the point.
```suggestion
    """Easter conversation topics."""
```